src/pages/Register/Register.css

```diff
@@ -1 +1,57 @@
+.register-mainContainer{
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
 
+.register-cardContainer {
+    width: 45%;
+    height: 350px;
+}
+
+.register-cardContent {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+
+.register-inputContainer {
+    width: 100%;
+    height: 20px;
+    margin-top: 15px;
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+    justify-content: center;
+}
+
+.register-inputContainer1 {
+    width: 100%;
+    height: 200px;
+    margin-top: 15px;
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+    justify-content: center;
+}
+
+.register-description{
+    height: 80px;
+}
+
+.register-upload{
+    height: 200px
+}
+
+.register-buttonText {
+    margin-top: 0;
+    margin-bottom: 0;
+    font-size: 1.25rem;
+}
+
+button:hover {

```

Comment:
styles like these will affect all html button elements because you are styling with a tag selector not a class selector. Additionally you do not use any button tags in Register.js. If you meant this styling for the register button, you don't need this since the component already handles the hover affect (check Button.css).